src/MAUI/Maui.Samples/Samples/Symbology/FeatureLayerExtrusion/FeatureLayerExtrusion.xaml.cs

```diff
@@ -64,7 +64,8 @@ private async Task Initialize()
                 // Set the extrusion mode for the scene properties.
                 myRendererSceneProperties.ExtrusionMode = ExtrusionMode.AbsoluteHeight;
 
-                // Set the initial extrusion expression.
+                // Set the initial extrusion expression to the quotient of the 2007 population field, with a divisor of 10.
```

Comment:
This is a nitpick, but I would edit this to be  a single, more simplistic comment. Something like "Set the initial extrusion expression to  "2007 population divided by ten" to improve visibility".